models/statistic.go

```diff
@@ -23,20 +23,20 @@ type Statistic struct {
 func GetStatistic() (stats Statistic) {
 	stats.Counter.User = CountUsers()
 	stats.Counter.Org = CountOrganizations()
-	stats.Counter.PublicKey, _ = db.DefaultContext().Engine().Count(new(PublicKey))
+	stats.Counter.PublicKey, _ = db.GetEngine(db.DefaultContext).Count(new(PublicKey))
 	stats.Counter.Repo = CountRepositories(true)
-	stats.Counter.Watch, _ = db.DefaultContext().Engine().Count(new(Watch))
-	stats.Counter.Star, _ = db.DefaultContext().Engine().Count(new(Star))
-	stats.Counter.Action, _ = db.DefaultContext().Engine().Count(new(Action))
-	stats.Counter.Access, _ = db.DefaultContext().Engine().Count(new(Access))
+	stats.Counter.Watch, _ = db.GetEngine(db.DefaultContext).Count(new(Watch))
+	stats.Counter.Star, _ = db.GetEngine(db.DefaultContext).Count(new(Star))
+	stats.Counter.Action, _ = db.GetEngine(db.DefaultContext).Count(new(Action))
+	stats.Counter.Access, _ = db.GetEngine(db.DefaultContext).Count(new(Access))
```

Comment:
Could introduce a variable to reduce the calls.

Reply:
As you might imagine, the code was a result of a mass find and replace.

Reply:
Sure, but could be changed while working at it.